### PR TITLE
Fix CSS of alert messages

### LINKED ISSUE #218: CSS for error in form UI
**Is your feature request related to a problem? Please describe.**
We need to add the css for the errors in the form UI.

**Describe the solution you'd like**
Implementation of attached UI for errors. 

**Additional context**
Refer [UI Toolkit](https://www.figma.com/file/BrP5RowC7va50HyETZQUDA/Ignitus-Design?node-id=0%3A1) for User-Interface and other style-related issues.

**Expected-Interface**
screenshot from 2018-12-31 23-51-24

### PR DESCRIPTION
**Pull request by arc9693**
- Modification of CSS of error/alert messages

Fixes #218 

### Description
- For email validation,added one more state variable called invalidEmail.
- Changed positioning and color of error/alert messages

### Screenshots :
screenshot from 2019-03-03 00-00-20
screenshot from 2019-03-03 00-01-11
screenshot from 2019-03-03 00-04-24

### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)

### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Paarmita** [comment]
Hey @arc9693, Can you please see the designs and improve the implementation to it like the color, margins, buttons style, etc, then you are good to go 👍

**Comment by arc9693** [comment]
@Paarmita  Okay. Looking into it.

**Comment by arc9693** [comment]
@Paarmita @divyanshu-rawat  Hi i have made changes in the css. The force push was because of rebasing and squashing of commits.Please review.
Screenshot from 2019-04-13 03-39-50
Screenshot from 2019-04-13 03-15-39

File: src/ignitus-StudentLogin/Components/Login.js [hunk 1]
```diff
@@ -1,45 +1,58 @@
...
+      }
+    } 
+
+    this.props.logInRequest(email, password);
+    this.setState({ email: "", password: "", emptymessage: false });
+
   }
 
   render() {
+    console.log('msg', this.props.studentLoginData)
```
Comment (Paarmita) [inline src/ignitus-StudentLogin/Components/Login.js]: Can you please remove the console statements from the commit

File: src/ignitus-StudentSignUp/Components/Signup.js [hunk 2]
```diff
@@ -1,196 +1,181 @@
...
+              </div>
+            </form>
           </div>
-          )}
         </div>
       </div>
+    </div>
     );
   }
 }
```
Comment (Paarmita) [inline src/ignitus-StudentSignUp/Components/Signup.js]: Why did you removed these prop-types?

Comment (arc9693) [reply src/ignitus-StudentSignUp/Components/Signup.js]: I am sorry I didn't get you. I shifted the alert messages from there to support their current position.

File: src/ignitus-StudentLogin/Components/Login.js (hunk 1 above)
Comment (arc9693) [reply src/ignitus-StudentLogin/Components/Login.js]: Sure

File: src/ignitus-StudentSignUp/Components/Signup.js (hunk 2 above)
Comment (Paarmita) [reply src/ignitus-StudentSignUp/Components/Signup.js]: No, you removed the prop-types. Can you please add them again?

**Comment by arc9693** [comment]
@Paarmita  I have resolved changes requested. Kindly check.

**Pull request merged**